var/spack/repos/builtin/packages/slate/package.py

```diff
@@ -16,18 +16,17 @@ class Slate(Package):
     solvers."""
 
     homepage = "https://icl.utk.edu/slate/"
-    hg      = "https://bitbucket.org/icl/slate"
-    maintainers = ['G-Ragghianti']
+    url      = "https://bitbucket.org/icl/slate"
```

Comment:
Why not just put `git = "https://bitbucket.org/icl/slate"`, then you don't need the `git` keyword below for the individual versions.

Reply:
OK that works.  Done.